src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -400,18 +402,17 @@ export default {
 			}
 		},
 		// This is used to decide which outer element type to use
-		// li or router-link
 		navElement() {
 			if (this.to) {
 				return {
-					is: 'router-link',
-					tag: 'li',
+					is: 'div',
```

Comment:
This needs to stay router-link.
```suggestion
					is: 'router-link',
```